theme/js/dataset-recommendations.js

```diff
@@ -33,6 +33,8 @@ function addWidgetScript() {
 
 function loadDatasets() {
     udataScript.loadDatasets();
+    const recoParent = document.getElementById('dataset-recommendations');
```

Comment:
Why not here ? https://github.com/abulte/udata-gouvfr/blob/9a29430d13c91dd42873dc5d734c13eb264decda/theme/js/dataset-recommendations.js#L48

Reply:
I think the callback from the script is really the latest step we can find in the code. `addRecos()` should be pretty fast. I may be wring, I did not measure my hypothesis.

Reply:
But it gives visually good results compared to before (it *was* in the fetch success).